<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>LTC Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="UI.qrc">
    <normaloff>:/wav.png</normaloff>:/wav.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="5">
       <widget class="QCheckBox" name="checkDropFrame">
        <property name="text">
         <string>Drop Frame</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="sbToSecs">
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QSpinBox" name="sbToFrames">
        <property name="maximum">
         <number>29</number>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QComboBox" name="comboFrameRate">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>24 fps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25 fps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>29.97 fps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30 fps</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="sbToHours">
        <property name="maximum">
         <number>23</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>To</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="sbFromSecs">
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="sbToMins">
        <property name="maximum">
         <number>59</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>From</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="sbFromMins">
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sbFromHours">
        <property name="maximum">
         <number>23</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="sbFromFrames">
        <property name="maximum">
         <number>29</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Hours</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Minutes</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Seconds</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Frames</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBitRate">
        <property name="currentText">
         <string>8-bit</string>
        </property>
        <item>
         <property name="text">
          <string>8-bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16-bit</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Audio Format</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="comboSampleRate">
        <item>
         <property name="text">
          <string>44.1 kHz</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>48.0 kHz</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboSoundMode">
        <item>
         <property name="text">
          <string>Mono</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Stereo</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="buttonSave">
        <property name="text">
         <string>Save as WAV</string>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>sbFromHours</tabstop>
  <tabstop>sbFromMins</tabstop>
  <tabstop>sbFromSecs</tabstop>
  <tabstop>sbFromFrames</tabstop>
  <tabstop>sbToHours</tabstop>
  <tabstop>sbToMins</tabstop>
  <tabstop>sbToSecs</tabstop>
  <tabstop>sbToFrames</tabstop>
  <tabstop>comboFrameRate</tabstop>
  <tabstop>checkDropFrame</tabstop>
  <tabstop>comboSoundMode</tabstop>
  <tabstop>comboBitRate</tabstop>
  <tabstop>comboSampleRate</tabstop>
  <tabstop>buttonSave</tabstop>
 </tabstops>
 <resources>
  <include location="UI.qrc"/>
 </resources>
 <connections/>
</ui>
